br/backup-storage-azblob.md

```diff
@@ -57,107 +96,60 @@ This section shows backing up data to `cool tier`, that is, the access tier of t
 
 #### Restore
 
-When restoring data using Azure AD, you need to specify `account-name`. You can specify it in two ways:
+Based on the way `account-name` is specified, you can restore data either using URLs or command-line parameters:
 
-- Write the parameter information in URL parameters:
+- `account-name` is specified in URL parameters:
 
     ```
     tiup br restore db --db test -u 127.0.0.1:2379 -s 'azure://test/t1?account-name=devstoreaccount1'
     ```
 
-- Write the parameter information in command-line parameters:
+- `account-name` is specified in command-line parameters:
 
     ```
     tiup br restore db --db test -u 127.0.0.1:2379 -s 'azure://test/t1?' --azblob.account-name=devstoreaccount1
     ```
 
 ### Method 2: Back up and restore using an access key (easy)
 
-#### Back up
-
-When backing up data using an access key, you need to specify `account-name`, `account-key`, and `access-tier`. If `access-tier` is not set (the value is empty), the value is `Hot` by default.
+Compared with backing up and restoring data using Azure AD, backup and restoration using an access key is easier because you do not need to configure environment variables. Other steps are similar to those using Azure AD.
```

Comment:
```suggestion
Compared with backing up and restoring data using Azure AD, backup and restoration using an access key is easier because you do not need to configure environment variables. Other steps are similar to those of using Azure AD.
```